src/mongo/db/storage/mmap_v1/mmap_windows.cpp

```diff
@@ -184,8 +184,8 @@ void* MemoryMappedFile::map(const char* filenameIn, unsigned long long& length)
     strncpy(filename, filenameIn, 255);
     filename[255] = 0;
     {
-        size_t len = strlen(filename);
-        for (size_t i = len - 1; i >= 0; i--) {
+        int len = strlen(filename);
```

Comment:
Yes I understand that. My point is: what does a zero length filename mean overall to this function. Is it sensible? Or should it be rejected earlier, in which case we would know that strlen would return >0, so use of size_t would be correct both for capturing the return value of strlen, and for iterating.